docs/contributing.md

```diff
@@ -2,15 +2,40 @@
 
 ## Tech Stack
 
-The following is the list of major technologies we are currently using:
+These are the major technologies we are using. Familiarity with most of these will make it easier to contribute.
 
-- Typescript
-- React
-- Tailwindcss
-- Next.js
-- Postgresql
-- Prisma
+**Languages:** typescript
 
-## Getting Started
+**Build tools:** npm, nx
 
-You can set up your development environment by following [these steps](./setup.md).
+**Web client:** react, tailwind
+
+**API server:** node, nextjs, prisma, postgresql
+
+## Local Development
```

Comment:
It would be good to have a note about recommended terminals in each OS. Is that part of a future issue?

Reply:
I don't think the terminal should matter too much. It should work for sure in bash (including git bash) and zsh. I'm not sure what you are using. I added a note to that effect.

Reply:
PowerShell wasn't working well at all, so I switched to Git bash and it works much better.